cmd/object-api-interface.go

```diff
@@ -90,7 +90,7 @@ type ObjectLayer interface {
 	HealBucket(ctx context.Context, bucket string, dryRun, remove bool) (madmin.HealResultItem, error)
 	HealObject(ctx context.Context, bucket, object string, dryRun, remove bool) (madmin.HealResultItem, error)
 	ListBucketsHeal(ctx context.Context) (buckets []BucketInfo, err error)
-	ListObjectsHeal(ctx context.Context, bucket, prefix, marker, delimiter string, maxKeys int) (ListObjectsInfo, error)
+	HealObjects(ctx context.Context, bucket, prefix string, healObjectFn func(string, string) error) error
```

Comment:
It looks like we can come up with a better name, like ApplyFnOnObjects() since HealObjects only applies function on objects that it can find

Reply:
Really ?